doc/contributing.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to cardano-node-tests repo
+
+First off, thanks for taking the time to contribute!
+
+The following is a set of guidelines for contributing to cardano-node-tests.
+These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes
+to this document in a pull request.
+
+## How Can I Contribute?
+
+### Reporting an Issue
+
+This section guides you through submitting an issue report for cardano-node-tests. Following these guidelines helps maintainers and the community understand your report, reproduce the behavior, and find related reports.
+
+Before creating bug reports, please check [this list of existing issues](https://github.com/input-output-hk/cardano-node-tests/issues) as you might find out that you don't need to create one. When you are creating a bug report, please [include as many details as possible](#How Do I Submit A (Good) Issue Report).
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+#### How Do I Submit A (Good) Issue Report
+
+Explain the problem and include additional details to help maintainers reproduce the problem:
+
+- **Use a clear and descriptive title** for the issue to identify the problem.
+- **Describe the exact steps which reproduce the problem** in as many details as possible. When listing steps, **don't just say what you did, but explain how you did it**.
+- **Provide specific examples to demonstrate the steps**. Include links to files or GitHub projects, or copy/pasteable snippets, which you use in those examples. If you're providing snippets in the issue, use [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+- **Describe the behavior you observed after following the steps** and point out what exactly is the problem with that behavior.
+- **Explain which behavior you expected to see instead and why.**
+- **Which version of cardano-node & cardano-node-tests are you using?**
+
+
+### Your First Code Contribution
+
+#### Pull Requests
+
+The process described here has several goals:
+
+- Maintain cardano-node-tests' quality
+- Fix problems that are important to users
+- Engage the community in working toward the best possible Cardano System Tests
+- Enable a sustainable system for cardano-node-tests' maintainers to review contributions
+
+Please follow these steps to have your contribution considered by the maintainers:
+
+1. Follow all instructions in [the template](PULL_REQUEST_TEMPLATE.md)
```

Comment:
PULL_REQUEST_TEMPLATE.md file doesn't exist. if it something that is to add later we should put a note or remove the link, otherwise will show a 404 error and its not a very good UX.

Reply:
makes sense; I removed the link and added a TODO note